packages/@pollyjs/adapter-node-http/tests/node/integration/adapter-test.js

```diff
@@ -21,4 +82,16 @@ describe('Integration | Node Http Adapter | Node', function() {
   setupPolly.afterEach();
 
   adapterTests();
+
+  binaryDownloadTest(
+    'should be able to fetch binary content with http',
+    http,
+    'http://via.placeholder.com/150/92c952'
+  );
+
+  binaryDownloadTest(
```

Comment:
I also want to add upload tests, i'm not sure where can i upload them. Should i create an express server in tests just to test uploading ?

Reply:
Yeah, you can just modify `tests/middleware.js` and add your own upload route.